packages/opal/opal.0.1.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Linjie Ding <i@pyroc.at>"
+authors: "Linjie Ding <i@pyroc.at>"
+homepage: "https://github.com/pyrocat101/opal"
+bug-reports: "https://github.com/pyrocat101/opal/issues"
+license: "MIT"
+dev-repo: "https://github.com/pyrocat101/opal.git"
+build: ["%{make}" "ncl" "bcl"]
```

Comment:
The build is failing because the variable splice is missing a `%` here and in the `install` section; it should be

```
build: ["%{make}%" "ncl" "bcl"]
```

But it would be simpler to write the `make` directly at the top level of the command, like this:

```
build: [make "ncl" "bcl"]
```